test.rb

```diff
@@ -1,8 +1,9 @@
 require "json"
 require "faraday"
+require "rspec"
```

Comment:
A me funziona anche senza il require

Reply:
se lo invoca con rspec sì, se lo esegue con ruby fallisce perché RSpec non è definito. Potrebbe essere una buona idea controllare se è stato invocato con ruby e, in caso, mostrare un messaggio di errore

Reply:
beh ma io direi di invocarlo con rspec non con ruby